app/templates/views/guidance/using-notify/daily-limits.html

```diff
@@ -50,14 +50,14 @@ <h3 class="heading-medium">Change the international text message limit</h3>
  <li>how many international text messages your service needs to send each day</li>
  <li>any costs to your service if there is fraud</li>
  </ul>
-<p class="govuk-body">If you do not need to send international text messages, you can turn this off in <b class="govuk-!-font-weight-bold">Settings</b>.</p>
+<p class="govuk-body">If you do not need to send international text messages, you can turn this off on the settings page.</b>.</p>
```

Comment:
This will add a link to the user’s settings page, if they are signed in and we can tell which service they are using
```suggestion
<p class="govuk-body">If you do not need to send international text messages, you can turn this off on the {{ service_link(current_service, 'main.service_settings', 'settings') }} page.</b>.</p>
```